Modules/Microsoft365DSC/Examples/Resources/AADAuthenticationMethodPolicyAuthenticator/1-AADAuthenticationMethodPolicyAuthenticator-Example.ps1

```diff
@@ -0,0 +1,86 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [PSCredential]
+        $Credscredential
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+
+    Node localhost
+    {
+        AADAuthenticationMethodPolicyAuthenticator "AADAuthenticationMethodPolicyAuthenticator-MicrosoftAuthenticator"
+        {
+            ApplicationId         = $ConfigurationData.NonNodeData.ApplicationId;
```

Comment:
If you are using the Service Principal Auth flow in your example, please remove the mandatory CredsCredential parameter